app/components/create_card/index.js

```diff
@@ -15,6 +15,12 @@ import KeyboardSpacer from 'react-native-keyboard-spacer';
 import {RSAKeychain, RSA} from 'react-native-rsa-native';
```

Comment:
but when merged, make sure that "react-native-rsa-native" changed back to "react-native-rsa"